src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,193 @@
+# Common data types and metadata
+
+## Common file level metadata fields
+
+Each derivative data file SHOULD be described by a JSON file provided as a sidecar
```

Comment:
Accepted suggestion in https://github.com/bids-standard/bids-specification/pull/488#discussion_r433280513.